src/ime.h

```diff
@@ -2,6 +2,11 @@
 #ifndef IME_H
 #define IME_H
 
+#if defined(_WIN32)
+#   include <windows.h>
+#   include <imm.h>
+#endif
+
```

Comment:
This can probably be moved to `ime.cpp` to reduce include dependency, since `ime.h` doesn't contain anything Windows-specific.